hangman/src/App.js

```diff
@@ -0,0 +1,223 @@
+import React, { Component } from "react"
+import logo from "./logo.svg"
+import Solution from "./components/Solution"
+import Score from "./components/Score"
+import Letters from "./components/Letters"
+import StartGame from "./components/startGame/StartGame"
+import EndGame from "./components/endGame/EndGame"
+import EndLevel from "./components/endLevel/EndLevel"
+import Hangman from "./components/Hangman"
+import "./App.css"
+
+class App extends Component {
+  constructor() {
+    super()
+    this.state = {
+      letterStatus: this.generateLetterStatuses(),
+      solutions: [
+        {
+          level: 1,
+          word: "Soup",
+          hint: "I'm a food but also a drink",
+        },
+        {
+          level: 2,
+          word: "Genie",
+          hint: "I'm a magician trapped in a bottle",
+        },
+        {
+          level: 3,
+          word: "Moose",
+          hint: "I'm an animal with hair products",
+        },
+        {
+          level: 4,
+          word: "Sponge",
+          hint: "I'm full of holes but I still hold water",
+        },
+        {
+          level: 5,
+          word: "Future",
+          hint: "I'm always in front of you, but I cannot be seen",
+        },
+        {
+          level: 6,
+          word: "Echo",
+          hint: "What can’t talk but will reply when spoken to",
+        },
+        {
+          level: 7,
+          word: "Yarn",
+          hint:
+            "I’m found in socks, scarves and mittens; and often in the paws of playful kittens",
+        },
+        {
+          level: 8,
+          word: "Window",
+          hint: "What invention lets you look right through a wall?",
+        },
+        {
+          level: 9,
+          word: "Secret",
+          hint:
+            "If you’ve got me, you mustn't share me; if you share me, you haven’t kept me. What am I?",
+        },
+        {
+          level: 10,
+          word: "Name",
+          hint:
+            "It belongs to you, but other people use it more than you do. What is it?",
+        },
+      ],
+      activeSolution: {},
+      score: 100,
+      gameStatus: {
+        isGameStarted: false,
+        isGameOver: false,
+        isPlayerWon: false,
+      },
+    }
+  }
+  generateLetterStatuses() {
+    let letterStatus = {}
+    for (let i = 65; i < 91; i++) {
+      letterStatus[String.fromCharCode(i)] = false
+    }
+    return letterStatus
+  }
+  resetStates = {
+    letterStatus: async () => {
+      let resetLetterStatus = this.generateLetterStatuses()
+      this.setState({ letterStatus: resetLetterStatus })
+    },
+    gameStatus: async () => {
+      let resetGameStatus = this.state.gameStatus
+      resetGameStatus.isGameOver = false
+      resetGameStatus.isPlayerWon = false
+      resetGameStatus.isGameStarted = true
+      this.setState({ gameStatus: resetGameStatus })
+    },
+    score: async () => {
+      this.setState({ score: 100 })
```

Comment:
You declaring the method as synchronized methods but not using any `await`, what should it be `async`?